tests/support/rally.test.mjs

```diff
@@ -63,18 +63,42 @@ describe('Rally', function () {
 
       assert.rejects(
         this.rally._checkRallyCore(),
-        {message: "Rally._checkRallyCore - core addon not found"}
+        { message: "Rally._checkRallyCore - core addon not found" }
       );
     });
 
-    it('must succeede if the core addon is installed', async function () {
+    it('must succeed if the core addon is installed', async function () {
       chrome.runtime.sendMessage.yields(
-        {type: "core-check-response", data: {enrolled: true}});
+        { type: "core-check-response", data: { enrolled: true } });
 
       await this.rally._checkRallyCore();
     });
   });
 
+  describe('_pause()', function () {
+    it('pauses when receiving message', async function () {
+      this.rally._handleExternalMessage("pause", "rally-core@mozilla.org");
+
+      assert.equal(this.rally._state = "paused");
+    });
+  });
+
+  describe('_resume()', function () {
+    it('resumes when receiving message', async function () {
+      this.rally._handleExternalMessage("resume", "rally-core@mozilla.org");
+
+      assert.equal(this.rally._state = "running");
+    });
+  });
+
+  /*
+  describe('addStateListener()', function () {
```

Comment:
Don't forget to remove :)

Reply:
Not needed now that we're doing it as part of initialize, I guess :)